<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandaloneMeshViewer</class>
 <widget class="QMainWindow" name="StandaloneMeshViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>543</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="windowTitle">
    <string>Standalone Mesh Viewer</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MeshViewWidget" name="viewerWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="annotationTab">
        <attribute name="title">
         <string>Annotation</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="PartonomyWidget" name="partonomyWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="annotationTools">
           <item>
            <widget class="QPushButton" name="saveAnnotation">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Save New Annotation</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="currentAnnotationsGroupBox">
           <property name="title">
            <string>Current Annotations</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QListView" name="currentAnnotationsView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="annotationButtonsLayout">
              <item>
               <widget class="QPushButton" name="removeAnnotationButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editAnnotationButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="debugTab">
        <attribute name="title">
         <string>Debug</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Property Color Transfer Tuning</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="windowLevelButton">
              <property name="text">
               <string>Window-level scale</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TweakParamWidget" name="tweakParams" native="true"/>
            </item>
            <item>
             <widget class="QRadioButton" name="minMaxButton">
              <property name="text">
               <string>Min-max scale</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TweakParamWidget" name="minMaxParams" native="true"/>
            </item>
            <item>
             <widget class="QCheckBox" name="invertScaleCheckBox">
              <property name="text">
               <string>Invert</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="vtkTools">
              <item>
               <widget class="QLabel" name="colorSeriesLabel">
                <property name="text">
                 <string>Color Series #:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="colorSeriesSelector">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="applyButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Mesh"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Selection"/>
    <addaction name="actionSave_Selection"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Geometric_Property"/>
    <addaction name="actionLoad_Annotations"/>
    <addaction name="actionLoad_Facets"/>
    <addaction name="actionLoad_Bounding_Box"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Vocabulary"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad_Mesh">
   <property name="text">
    <string>Load Mesh</string>
   </property>
  </action>
  <action name="actionLoad_Geometric_Property">
   <property name="text">
    <string>Load Geometric Property</string>
   </property>
  </action>
  <action name="actionLoad_Annotations">
   <property name="text">
    <string>Load Annotations</string>
   </property>
  </action>
  <action name="actionLoad_Facets">
   <property name="text">
    <string>Load Facets</string>
   </property>
  </action>
  <action name="actionLoad_Bounding_Box">
   <property name="text">
    <string>Load Bounding Box</string>
   </property>
  </action>
  <action name="actionLoad_Selection">
   <property name="text">
    <string>Load Selection</string>
   </property>
  </action>
  <action name="actionSave_Selection">
   <property name="text">
    <string>Save Selection</string>
   </property>
  </action>
  <action name="actionLoad_Vocabulary">
   <property name="text">
    <string>Load Vocabulary</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeshViewWidget</class>
   <extends>QWidget</extends>
   <header>core/meshviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PartonomyWidget</class>
   <extends>QWidget</extends>
   <header>partonomywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TweakParamWidget</class>
   <extends>QWidget</extends>
   <header>tweakparamwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
